apimonitor/monitor/health.py

```
@@ -0,0 +1,162 @@
+            logger.warning(
+                {
+                    'service': service.name,
+                    'url': url,
+                    'error': 'timeout',
+                    'attempt': attempt + 1,
+                    'ts': time.time()
+                }
+            )
+            
+        except NetworkError as e:
+            last_error = f"Network error: {str(e)}"
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)